.gitlab/packaging/deb.yml

```diff
@@ -22,108 +22,95 @@
   cache:
     - !reference [.cache_omnibus_ruby_deps, cache]
 
+.package_deb_x86:
+  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-buildimages/deb_x64$DATADOG_AGENT_BUILDIMAGES_SUFFIX:$DATADOG_AGENT_BUILDIMAGES
+  tags: ["arch:amd64"]
+  variables:
+    DD_PKG_ARCH: "x86_64"
+    PACKAGE_ARCH: amd64
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏I fear that later someone might forget the `"` while modifying this, isn't it possible to get rid of them everywhere ?